data/mods/blazemod/blaze_blob.json

```diff
@@ -860,10 +878,19 @@
     "material": [ "flesh" ],
     "symbol": ";",
     "color": "green",
-    "contains": 60,
+    "contains": 80,
     "seals": true,
     "watertight": true
   },
+  {
+    "id": "oozle_tank_large",
+    "type": "CONTAINER",
+    "name": "oozing tank",
+    "name_plural": "oozing tanks",
+    "copy-from": "oozle_tank",
+    "volume": 320,
+    "contains": 320
```

Comment:
Needs to mass more than an oozle_tank - at least twice as much, since it has roughly 10x the volume and 4x the surface area.